resources/css/_general.css

```
@@ -405,3 +405,20 @@ input.qr-code-amount[type="number"] {
         background-image: url("/resources/images/home/arrows-dim.svg") !important;
     }
 }
+
+/* Code Block */
+/* Note: I've copied most of the `.custom-scroll` class here. Some of the existing 
+   class makes the scrollbar "thick", like it would be on any other website. */
+.code-block-custom-scroll::-webkit-scrollbar {
+    @apply rounded-lg;
+    width: 6px;
+    height: 4px;
```

```suggestion
    @apply rounded-lg w-1.5 h-1;
```

would this work too?